### ISSUE TITLE
About AAC and WebRTC

### ISSUE THREAD

**Issue by AngellusMortis** [AlexxIT/WebRTC#110]
Why doesn't the WebRTC player support AAC audio? I am using Unifi Protect cameras and as far as I can tell, they have AAC audio and the Unifi Protect Web app uses WebRTC with audio just fine. 

As far as I can tell from [MDN](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/WebRTC_codecs) and [caniuse](https://caniuse.com/aac), it looks like it should work. Browsers via WebRTC are required to suppose Opus, PMCA and PMCU, but they can support other codecs as well and all of the major browsers also suppose AAC.

**Comment by AlexxIT** [comment]
WebRTC have not support AAC. I think Unifi users another technology
https://developer.mozilla.org/ru/docs/Web/Media/Formats/WebRTC_codecs

**Comment by AngellusMortis** [comment]
That is the page I linked. it explicitly states that those are just the codecs that are _required_ to be implemented, not a comprehensive list of all of the ones that _can_ be supported. 

> This guide reviews the codecs that browsers are required to implement as well as other codecs that some or all browsers support for WebRTC.

**Comment by SeBsZ** [comment]
I used chrome://webrtc-internals to collect the information about what Unifi uses. However, I don't see anything about audio or AAC in here. @AlexxIT can understand this data and see if WebRTC+AAC is used? I'm having the same issue as @AngellusMortis where I don't get any audio when WebRTC is used, but I do get audio when I set webrtc: false, but then I get really bad video.
Many thanks for your help

[webrtc_internals_dump.txt](https://github.com/AlexxIT/WebRTC/files/7976990/webrtc_internals_dump.txt)
